src/redist/redist.csproj

```diff
@@ -154,18 +154,6 @@
           Condition=" '$(DISABLE_CROSSGEN)' == '' "
           AfterTargets="PublishMSBuildExtensions">
     <ItemGroup>
-      <NETCore10Assemblies Include="$(PublishDir)/System.ComponentModel.Primitives.dll;
```

Comment:
@eerhardt In latest TestPlatform.CLI nuget package, doesn't have any NetCore10Assemblies.